templates/remove.j2

```diff
@@ -1,79 +1,73 @@
 {% endif %}
 
-#=================================================
-# GENERIC START
-#=================================================
-# IMPORT GENERIC HELPERS
-#=================================================
 source _common.sh
 source /usr/share/yunohost/helpers
 
 {% if data.generator_mode == 'tutorial' -%}
-# Settings are automatically loaded as bash variables
-# in every app script context, therefore typically these will exist:
-# - $domain
-# - $path
-# - $language
-# - $install_dir
-# - $port
-# ...
-
-# For remove operations :
-# - the core will deprovision every resource defined in the manifest **after** this script is ran
-# this includes removing the install directory, and data directory (if --purge was used)
+### Settings are automatically loaded as bash variables
+### in every app script context, therefore typically these will exist:
+### - $domain
+### - $path
+### - $language
+### - $install_dir
+### - $port
+### ...
+
+### For remove operations:
+### - the core will deprovision every resource defined in the manifest **after** this script is ran
+### this includes removing the install directory, and data directory (if --purge was used)
 {% endif %}
 
 #=================================================
 # REMOVE SYSTEM CONFIGURATIONS
 #=================================================
-# REMOVE SYSTEMD SERVICE
-#=================================================
-ynh_script_progression --message="Removing system configurations related to $app..."
+ynh_script_progression "Removing system configurations related to $app..."
 
 {% if data.generator_mode == 'tutorial' -%}
 # This should be a symetric version of what happens in the install script
 {% endif %}
 
+{% if data.use_fail2ban -%}
+ynh_config_remove_fail2ban
+{% endif %}
+
+{% if data.use_logrotate -%}
+ynh_config_remove_logrotate
+{% endif %}
+
 {% if data.main_technology not in ["php", "none"] -%}
 # Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
-if ynh_exec_warn_less yunohost service status $app >/dev/null
-then
-	ynh_script_progression --message="Removing $app service integration..."
+if ynh_hide_warnings yunohost service status $app >/dev/null; then
 	yunohost service remove $app
 fi
 
-ynh_remove_systemd_config
+ynh_config_remove_systemd
 {% endif %}
 
-ynh_remove_nginx_config
+ynh_config_remove_nginx
 
 {% if data.main_technology == "php" -%}
-ynh_remove_fpm_config
+ynh_config_remove_phpfpm
 {% endif %}
 
-{% if data.use_logrotate -%}
-ynh_remove_logrotate
-{% endif %}
-
-{% if data.use_fail2ban -%}
-ynh_remove_fail2ban_config
+{% if data.generator_mode == 'tutorial' -%}
+# Remove other various files specific to the app... such as:
 {% endif %}
 
-# Remove other various files specific to the app... such as :
 {% if data.use_cron -%}
-ynh_secure_remove --file="/etc/cron.d/$app"
+ynh_safe_rm "/etc/cron.d/$app"
 {% endif %}
 
-ynh_secure_remove --file="/var/log/$app"
+ynh_safe_rm "/etc/$app"
```

Comment:
I dunno if we should keep this as default ... not that many apps have stuff stored in `/etc/$app` ?